tests/create_from_yaml.py

```diff
@@ -0,0 +1,197 @@
+import concurrent.futures
```

Comment:
This lookslike a forked version of https://github.com/kubernetes-client/python/blob/master/kubernetes/utils/create_from_yaml.py I would like to prevent using an internal fork this will only result in pain. Please use the  `create_from_yaml` function and add an extra method that wait until thee deployments are ready. You can also try to provide an upstream patch (e.g. create an issue and ask if this feature is wanted).